### PR TITLE
[IMP] attachment_s3

### LINKED ISSUE #215: attachment_s3: One directory per database
# Context

Using version 12.0 with Digital Ocean Spaces

# Steps to reproduce

* Setup the AWS environment variables
* Create a database "MASTER"

# Result

* Files are stored directly in /BUCKETNAME/

# Expected result

* Files are stored in /BUCKETNAME/MASTER/

### PR DESCRIPTION
**Pull request by max3903**
Closes #215

### CODE REVIEW

**Comment by max3903** [comment]
@guewen This is ready to be reviewed.

**Review by gustavovalverde — APPROVED** [review]

File: attachment_s3/models/ir_attachment.py [hunk 1]
```diff
@@ -46,53 +44,54 @@ def _get_s3_bucket(self, name=None):
...
+            host = "https://%s" % host
 
-        region_name = os.environ.get('AWS_REGION')
-        access_key = os.environ.get('AWS_ACCESS_KEY_ID')
-        secret_key = os.environ.get('AWS_SECRET_ACCESS_KEY')
-        bucket_name = name or os.environ.get('AWS_BUCKETNAME')
+        region_name = os.environ.get("AWS_REGION")
+        access_key = os.environ.get("AWS_ACCESS_KEY_ID")
+        secret_key = os.environ.get("AWS_SECRET_ACCESS_KEY")
+        bucket_name = name or os.environ.get("AWS_BUCKETNAME") + "-" + self.env.cr.dbname
```
Comment (yvaucher) [inline attachment_s3/models/ir_attachment.py:56]: Same concerns as on #218 

Isn't this a breaking change for existing instance with single database?

Plus, this give always a value to `bucket_name`

Which breaks the condition test on line 66

**Review by yvaucher — CHANGES_REQUESTED** [review]
It breaks the check on line 66

I guess there is no magic there, changing the bucket name computation force the users to rename their buckets?

**Comment by yvaucher** [comment]
Superseeded by https://github.com/camptocamp/odoo-cloud-platform/pull/227

**Pull request closed without merging**